astropy/uncertainty/core.py

```diff
@@ -220,6 +220,21 @@ def pdf_percentiles(self, percentile, **kwargs):
             return self.distribution._new_view(percs)
         else:
             return percs
+            
+    def pdf_asymstd(self):
+        """ 
+        a convenience function that computes the 50% percentile
+        value and the asymmetric uncertainties around that value.
+        ( p50, p84-p50, p50-p16).
```

Comment:
This would better given as (50%, +34%, -34%) as the `p` notation is not defined here.